packages/shared/src/services/bigquery/BigQueryProvider.test.ts

```diff
@@ -0,0 +1,62 @@
+import { expect, mockFn, mockObject } from 'earl'
+
+import { BigQueryProvider } from './BigQueryProvider'
+import { BigQueryWrapper } from './BigQueryWrapper'
+
+describe(BigQueryProvider.name, () => {
+  describe(BigQueryProvider.prototype.query.name, () => {
```

Comment:
write one more test that will check whether it returns the result of the .getQueryResults